Add VSCode launch configuration for test app

fosterseth opened the pull request:
Adds a launch configuration for test app

This will launch the test app server via VSCode so that develops can set debug points and step through code using VSCode.

Additional changes:

bootstrap.sh can be called with args

./bootstrap.sh startpostgres

this allows the launch configuration to start the postgres server separately from runserver.

./bootstrap.sh without args runs exactly as it did before.

![image](https://github.com/ansible/django-ansible-base/assets/8745776/56ba8bb8-cc7b-4a47-b169-70500d734ebf)

AlanCoding commented on .vscode/tasks.json:
```diff
@@ -0,0 +1,10 @@
+      {
+        "label": "bootstrap",
+        "type": "shell",
+        "command": "${workspaceFolder}/test_app/scripts/bootstrap.sh startpostgres",
```
So this is like a separate button you click in VSCode? I get why you need to give it your python interpreter and have it wrap it or whatever to do it in debug mode. But it seems a lot less clear how to manage a background container.

AlanCoding replied:
Maybe it would be easier to just let VSCode use sqlite3. We were until like this morning.

AlanCoding commented:
Otherwise this looks good to me. I just wonder if maintaining a postgres database at _this_ point is counter productive to the development experience. I never wanted it to be postgres-or-the-highway.

fosterseth committed:
Add VSCode launch configuration for test app

This will launch the test app server via VSCode
so that develops can set debug points and step
through code using VSCode.

As part of this, adds a sqlite3settings file so that
the server can run with sqlite3, rather than postgres

Signed-off-by: Seth Foster <fosterbseth@gmail.com>

fosterseth commented:
@AlanCoding made some changes

- uses sqlite3 instead of postgres
- no longer uses the bootstrap.sh script, just call a custom script made for vscode situations
